browser-tests/snapshot-tests/snapshot.spec.js

```diff
@@ -19,6 +19,13 @@ test("Snapshots", async ({ page }) => {
         }
         const screenshotFilename = `${pageName}-${language}${screenshotContext}.jpeg`;
         await page.goto(url);
+
+        // Open all summaries on the page
+        const allSummaries = await page.locator("summary").all();
+        for (const summary of allSummaries) {
+          await summary.click();
+        }
```

Comment:
Patrick pointed out on my accessibility tests PR that this may want to have a series of `waitFor` afterwards as suggested in the docs: https://playwright.dev/docs/accessibility-testing#configuring-axe-to-scan-a-specific-part-of-a-page

> / It is important to waitFor() the page to be in the desired
  // state *before* running analyze(). Otherwise, axe might not
  // find all the elements your test expects it to scan.

I've done it [here](https://github.com/govuk-one-login/ipv-core-front/pull/1575/files#:~:text=const%20allSummaryDetails%20%3D,%7D)

Reply:
That comment appears to be specific to Axe, but I guess there's no harm in it